### ISSUE TITLE
[Catalog] Publish Cloudnative Apps to Catalog

### ISSUE THREAD

**Issue by vishalvivekm** [meshery/meshery#12111]
#### Current Behavior
[Meshery Catalog](https://meshery.io/catalog) has a relatively small number of published designs.

#### Desired Behavior
The catalog should be populated with a variety of published designs.

#### Implementation
1. Import common sample apps from https://github.com/kubernetes/examples, or https://github.com/AdminTurnedDevOps/kubernetes-examples, or https://github.com/GoogleCloudPlatform/kubernetes-engine-samples as well as any other resources that can benefit both newcomers and seasoned professionals in the CNCF ecosystem.
2. WASM Filters are welcome too. Learn more about Filters management in Meshery at https://docs.meshery.io/tasks/filter-management.
3. Share these imported apps with the Meshery Catalog. Refer to https://docs.meshery.io/concepts/catalog.

Issues:
- [ ] 
- [ ]
- [ ]
- [ ] 
- [ ]
- [ ]
- [ ]

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by Tharanishwaran** [comment]
@vishalvivekm, I'd like to work in this issue. could you please assign it to me?

**Comment by shinigami-777** [comment]
@vishalvivekm Can you specify which categories need to be populated with published designs

**Comment by jamieplu** [comment]
@shinigami-777 all categories are welcome and all are needed.

**Comment by jamieplu** [comment]
I'm assigning to both of you, because there are many designs to be created and published.

**Comment by jamieplu** [comment]
Oh, I can’t assign.

**Comment by marblom007** [comment]
Gotcha covered.

**Comment by Tharanishwaran** [comment]
Hi @marblom007 and @jamieplu Thanks for assigning me! I'll start working on it and update you soon

**Comment by Tharanishwaran** [comment]
@vishalvivekm, @marblom007  @jamieplu  Is this task completed entirely through Meshery's tools (mesheryctl and UI) for importing and publishing apps to the catalog, or does it require a Pull Request to the repository?

**Comment by Tharanishwaran** [comment]
Thankyou @lekaf974

**Comment by laksh-krishna-sharma** [comment]
@leecalcote I'd like to work in this issue with you guidance. could you please assign it to me?

**Comment by jamieplu** [comment]
@Darshan174 👀

**Comment by 0xfabb** [comment]
@jamieplu  @marblom007  Hey,  i would like to work on the issue, i mean i've been looking at open issues and there are many issues where last comment is kind of done thingy but they are still open and so not sure whether they are open or i can contribute or not, however, i have forked cloned repo and setup locally, and searching for issue to work on it, could you please assign me this, or any other issue, i could work on and is unassigned?

**Comment by Darshan174** [comment]
Hey @0xfabb @laksh-krishna-sharma theres a bunch more issues opened of this type (to publish designs to catalog) labelled 'hactoberfest', feel free to ask for assignment and jump on publishing those designs.

**Comment by Sahitya0805** [comment]
Hi, I would like to work on this issue. Could you please assign it to me?

**Comment by ishwar170695** [comment]
This issue appears to serve as an epic/tracker. It may be helpful to update the tracking checklist with the catalog publication issues that have already been completed (for example, [#9289]([url](https://github.com/meshery/meshery/issues/9289))) and any remaining work items.

**Comment by marblom007** [comment]
@ishwar170695, yes, you're right. If you can directly edit the issue description, please do. Or please close this issue and open a copy with proper issue description.

**Comment by JaswanthRalph** [comment]
can I work on this if there is no one assigned to this issue?

**Comment by krisselannramos** [comment]
Hi everyone! I am a new contributor participating in the LFX mentorship onboarding. This issue looks like a great way to get started—could you please assign it to me? @vishalvivekm

**Comment by ishwar170695** [comment]
Hi @JaswanthRalph @krisselannramos, this issue is probably not the best one to start with. It was intended as an epic/tracker for multiple catalog publication efforts rather than a single, well-defined contribution, and it still needs to be cleaned up.

For getting started, I'd recommend looking for a more specific good first issue or asking in the [Meshery](https://mesheryio.slack.com) Community Slack for suggestions on currently active beginner-friendly issues.

**Comment by shantanu-1607** [comment]
hi @vishalvivekm , i would like to work on this issue , can i be assigned this? thanks.
